CPAC/registration/registration.py

```diff
@@ -2319,6 +2540,234 @@ def warp_timeseries_to_T1template(wf, cfg, strat_pool, pipe_num, opt=None):
     return (wf, outputs)
 
 
+def warp_timeseries_to_T1template_abcd(wf, cfg, strat_pool, pipe_num, opt=None):
```

Comment:
@sgiavasis what about this function name?

Reply:
I think that's fine.